MovieApp/MainActivity.cs

```diff
@@ -0,0 +1,516 @@
+            public double vote_average { get; set; }
+            public int vote_count { get; set; }
+            public int width { get; set; }
+        }
+
+        public class Poster
+        {
+            public double aspect_ratio { get; set; }
+            public string file_path { get; set; }
+            public int height { get; set; }
+            public string iso_639_1 { get; set; }
+            public double vote_average { get; set; }
+            public int vote_count { get; set; }
+            public int width { get; set; }
+        }
+
+        public class RootObjectImage
+        {
+            public int id { get; set; }
+            public List<Backdrop> backdrops { get; set; }
+            public List<Poster> posters { get; set; }
+        }
+
+
+
+        //Genre
+
+        public class Genre
+        {
+            public int id { get; set; }
+            public string name { get; set; }
+        }
+
+        public class RootObject
+        {
+            public List<Genre> genres { get; set; }
+        }
+
+        IList<Genero> listaGenero = new List<Genero>();
+        public class Movie
+        {
+            public int vote_count { get; set; }
+            public int id { get; set; }
+            public bool video { get; set; }
+            public double vote_average { get; set; }
+            public string title { get; set; }
+            public double popularity { get; set; }
+            public string poster_path { get; set; }
+            public string original_language { get; set; }
+            public string original_title { get; set; }
+            public List<int> genre_ids { get; set; }
+            public string backdrop_path { get; set; }
+            public bool adult { get; set; }
+            public string overview { get; set; }
+            public string release_date { get; set; }
+            public string maximum { get; set; }
+            public string minimum { get; set; }
+            public List<Movie> results { get; set; }
+            public int page { get; set; }
+            public int total_results { get; set; }
+            //public DateTime dates { get; set; }
+            public int total_pages { get; set; }
+        }
+        public class UpcomingMovies
+        {
+            public virtual int ID { get; set; }
+            public virtual string titulo { get; set; }
+            public virtual string descricao { get; set; }
+
+            public virtual string data { get; set; }
+            public virtual Image poster { get; set; }
+        }
+        public class GeneroJson
+        {
+            public virtual int id { get; set; }
+            public virtual string name { get; set; }
+        }
+        public class Genero
+        {
+            public virtual int idGenero { get; set; }
+            public virtual string genero { get; set; }
+        }
+
+        public void list_ItemClick(object sender, AdapterView.ItemClickEventArgs e)

```

Comment:
Keep the pattern through the code is also important. If you are using Camel case to name methods, stick with it.